build.yaml

```diff
@@ -1699,6 +1699,17 @@ libs:
   filegroups:
   - grpc_test_util_base
   secure: true
+- name: e_passthrough
```

Comment:
I don't have much context, but the `e_passthrough` name is not very descriptive. Is this a test or a library?
Should this be `engine_passthrough_test`?

Reply:
It is an OpenSSL engine. Not a gRPC test in itself. h2_ssl_cert_test will use this name to specify the private key. `e_passthrough` is following the standard notation of OpenSSL engines. But I can change it to something else if you prefer. Do you want it to be changed to about `engine_passthrough` ?

Reply:
I'd prefer a name that will be self-descriptive and doesn't require everyone looking up what it is when the see the name.`engine_passthrough` is better, but `openssl_engine_passthrough`/`ssl_engine_passthrough` is perhaps the best? 
As said, I don't have that much context here, so I don't want to dictate the name.

Reply:
Going to change to `engine_passthrough` as that is also the filename and other options are becoming too long.